### DISCUSSION TITLE
RE: date-fns.org opencollective sponsorship hasn't updated yet

### DISCUSSION THREAD

**Discussion by cydianow** [js-fns/date-fns#2051] [General] [answered]
Hi

Yesterday I subscribed to date-fns.org $50 montly sponsorship package via opencollective, And sent 2 messages about listing my website [Goread.io](https://goread.io/) on sponsors section. Still I didn't get any update?

Really appriciate if anyone can update me in on this

Thanks

**Comment by kossnocorp** [comment, marked as answer]
@cydianow hey! I appreciate your support. The sponsors' list update involves manual labor and doesn't happen automatically. I don't guarantee an immediate response, and this is not a place to discuss that. However, I did push an update, and your link will go live soon. Thanks!
